### PR TITLE
Church armour

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>
What essentially i did was use the armour code made by lamamaster and essentially re-use it by new sprites requested by Nayu with his requests.

Yes I TESTED it yes it WORKS in game and the armour kit works correctly.
The sprites aswell.

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Guidesu** [comment]
Oh and made the 100 PLASTEEL to print a hardsuit to 25 and upped the values of the other materials too

File: code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm [hunk 1]
```diff
@@ -86,6 +86,12 @@
 	icon_state = "neotheology_testament_u"
 	license = -1
 
+// Same as the other NT disk, minus the medical designs. Spawns in public access bioprinters.
```
Comment (Guidesu) [inline code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm:89]:
```suggestion

```

**Comment by Dissentbay** [comment]
looks like my work of balancing church armor got taken away from me 😌

**Comment by Noobetski** [comment]
you should speedmerge this... _now_

**Comment by cdb-is-not-good** [comment]
So what's the final price of the suit with the bioprinters adjustments?

**Comment by Catalaria** [comment]
What the fuck? Who's the moron that sprited these armours! They look so goddamn ugly, urgh! This sojourn's church spriter is a loser. I hate him

**Comment by cdb-is-not-good** [comment]
I checked the math myself. As I suspected, with how the bioprinter works this lowers the cost of the suit from 50 plasteel to 5. This breaks a pretty simple,  core tenet of balance- adjust by 30% at a time, not 300%

**Comment by Guidesu** [comment]
BEGONE MERGE CONFLICT

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -257,6 +257,56 @@
...
+	icon_state = "monomial_gusoku"
+
+/obj/item/clothing/suit/armor/vest/path/factorial
+	name = "Factorial powergarb"
+	desc = "Powergarbs are curious things made by the Nadezdha's church arms forces that are by no means made out of circuits, but rather have golem foundations. Such curious technology that almost only the church possess with great variety beyond xenobiology, what explains the EMP-Immune nature of the powergarbs even if, in fact, the lightweight clamping arms behind them are alive and are actually living golems that consented into becoming the protection of a Factorial. Well trained factorials are able to use such arms to lift themselves out of the ground for better combat support - but until now, only ERT forces are able to do such a thing."
+	icon_state = "factorial_powergarb"
+
+obj/item/clothing/suit/armor/vest/path/factorial2
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:305]:
```suggestion
/obj/item/clothing/suit/armor/vest/path/factorial2
```

**Pull request merged**